File: DailyChallenges/1438/main.c
#include<stdio.h>

int longestSubarray(int* nums, int numsSize, int limit) {
    int* a=&nums[0];
    printf("%d",*a);


}
int main(){

    return 0;
    int nums[4]={8,2,4,7};
    int numsize=sizeof(nums)/sizeof(nums[0]);

    int limit=4;
    longestSubarray(nums,numsize,limit);
}

File: DailyChallenges/1438/main.py
class Solution(object):
    def longestSubarray(self, nums, limit):
        subarray=[]
        for i in range(len(nums)):
            subarray.append(nums[i])
            diff=abs(nums[i]-subarray[i])
            print(diff,subarray)






solution=Solution()
nums=[8,2,4,7]
limit=4
solution.longestSubarray(nums,limit)
